### ISSUE TITLE
Groups and auth support?

### ISSUE THREAD

**Issue by poVoq** [miterion/ldap2carddav#10]
It seems like this creates a large public listing of all found LDAP users? 

But for syncing with mobile devices via Dav5x or similar it would be nice if there was a way to limit contacts to LDAP group membership and have a personalized login based on the LDAP credentials.

Thanks for considering this!

**Comment by miterion** [comment]
How would such a limit look like? Would it be that people can only see contacts they share a group with?

Regarding the personalized login: I don't plan to add authentication to this, since it is quite easy to use mod_ldap for these tasks.

**Comment by poVoq** [comment]
Yes only group shared group membership, but how would it know which groups to show if there isn't a personalized login?

**Comment by miterion** [comment]
You can extract the logged in user from the authentication method, so that would work from my perspective.
What does require some thinking however is what to do if a user is in multiple groups
